geode-management/src/main/java/org/apache/geode/management/internal/ClusterManagementClient.java

```diff
@@ -27,7 +27,6 @@
 import org.apache.geode.management.api.ClusterManagementResult;
 import org.apache.geode.management.api.ClusterManagementService;
 import org.apache.geode.management.api.RestfulEndpoint;
-import org.apache.geode.management.client.RestTemplateResponseErrorHandler;
 
 /**
  * Implementation of {@link ClusterManagementService} interface which represents the cluster
```

Comment:
For consistency, I'd prefer to see that all of the implementors of the CMS interface keep `ClusterManagementService` in their class name. Hence this class could be`ClientClusterManagmentService`